guides/deployments/postgres.md

```diff
@@ -88,14 +93,14 @@ Helm chart.
 To install Coder:
 
 ```console
-helm install coder coder/coder -n <your-coder-namespace> --version=<VERSION> -f current-values.yml
+helm install coder coder/coder --namespace=<your-coder-namespace> --version=<VERSION> --values=current-values.yml
 ```
 
 To upgrade Coder:
 
 ```console
-helm upgrade coder coder/coder -n <your-coder-namespace> --version=<VERSION> -f current-values.yml
+helm upgrade coder coder/coder --namespace=<your-coder-namespace> --version=<VERSION> --values=current-values.yml
 ```
 
-If this process is successful, you'll be able to access Coder using the external
-IP address of the ingress controller in your cluster.
+Once you complete this process, you'll be able to access Coder using the
```

Comment:
```suggestion
Once this process is completed, you'll be able to access Coder using the
```

Reply:
Say no to passive sentences, IMO :no_good: - will defer to @khorne3 on this one

Reply:
Yep, active voice. Thanks!